src/core/cm/imagemanager/itf/imagedecrypter.hpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2025 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CORE_CM_IMAGEMANAGER_ITF_IMAGEDECRYPTER_HPP_
+#define AOS_CORE_CM_IMAGEMANAGER_ITF_IMAGEDECRYPTER_HPP_
+
+#include <core/common/cloudprotocol/desiredstatus.hpp>
+#include <core/common/types/types.hpp>
+
+namespace aos::cm::imagemanager {
+
+/** @addtogroup cm Communication Manager
+ *  @{
+ */
+
+/**
+ * Image decrypter interface.
+ */
+class ImageDecrypterItf {
```

Comment:
Crypto helper itf should be used instead of this interface:

https://github.com/aosedge/aos_core_lib_cpp/pull/354

Reply:
Wait when this PR will be merged